org-formation/organization.yaml

```diff
@@ -560,3 +561,15 @@ Organization:
         AccountOwner: vijay.yadav@sagebase.org
         budget-alarm-threshold: 2000
         budget-alarm-threshold-email-recipient: aws.8dxfh53@sagebase.org
+  P9hFg7I5Account:
+    Type: OC::ORG::Account
+    Properties:
+      AccountName: org-sagebase-P9hFg7I5
```

Comment:
non-enforced account name convention is 7 chars

Reply:
oof, ok